src/components/elements/customUI/progressDots/ProgressDots.tsx

```diff
@@ -0,0 +1,77 @@
+/* Import Dependencies */
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import classNames from 'classnames';
+import { Row, Col } from 'react-bootstrap';
+
+/* Import Types */
+import { ProgressDot } from 'app/Types';
+
+/* Import Icons */
+import { faCircle } from '@fortawesome/free-solid-svg-icons';
+
+/* Import Components */
+import { Button } from '../CustomUI';
+
+
+/* Props Type */
+type Props = {
+    progressDots: ProgressDot[]
+};
+
+
+/**
+ * 
+ * @returns JSX Component
+ */
+const ProgressDots = (props: Props) => {
+    const { progressDots } = props;
+
+    return (
+        <div>
+            <Row>
+                {progressDots.map((progressDot, index) => {
+                    /* Class Names */
+                    const progressDotClass = classNames({
```

Comment:
did you mean to comment out this code?

Reply:
Fixed this in an upcoming PR already, but will remove it for this PR in a new commit

Reply:
Fixed this in an upcoming PR already, but will remove it for this PR in a new commit